docs/quickstarts/setup-clerk.mdx

```diff
@@ -1,59 +1,38 @@
 ---
-title: Set up Clerk
 description: Set up a new Clerk account and integrate it into a new application.
 ---
 
 # Set up your Clerk account
 
-## You will learn how to
-
-* Create a Clerk account
-* Create a new Clerk project
-
-**Estimated time** 1 cup of coffee
+Before you can start integrating Clerk into your application, you need to create a Clerk account and set up a new application in the Clerk Dashboard. This guide will walk you through those steps.
 
 <Callout type="info">
   Migrating from another platform?
 
-  You might want to [check out one of our migration guides](/docs/deployments/migrate-overview) to help
-  you move your data to Clerk.
+  You might want to check out one of the [migration guides](/docs/deployments/migrate-overview) to help you move your data to Clerk.
 </Callout>
 
 <Steps>
 
 ### Sign into Clerk
 
-[Create a Clerk account](https://dashboard.clerk.com/sign-up) or [sign into your Clerk dasbhoard](https://dashboard.clerk.com/).
+[Create a Clerk account](https://dashboard.clerk.com/sign-up) or [sign into your Clerk Dashboard](https://dashboard.clerk.com/).
 
 ### Create a Clerk application
 
-If you've just created an account for the first time, you'll be taken directly to our interactive authentication setup form.
-
-{/* Don't move ![] to a new line, otherwise MDX adds a <p> tag and it breaks A11Y */}
-
-<div style={{filter: "drop-shadow(rgba(0, 0, 0, 0.16) 0px 12px 24px)"}}>![An interactive builder to make your "<SignIn />" component truly yours, which lets you select your auth providers.](/docs/images/setup-clerk/create-sign-in-comp.png)</div>
-
-Otherwise, you'll be redirected to [your Clerk dashboard](https://dashboard.clerk.com/).
-
-To create a new app, press the **Add application** card to be taken to the interactive authentication setup form.
-
-{/* Don't move ![] to a new line, otherwise MDX adds a <p> tag and it breaks A11Y */}
-
-<div style={{filter: "drop-shadow(rgba(0, 0, 0, 0.16) 0px 12px 24px)", maxWidth: "400px"}}>![An "Add application" card on your dashboard](/docs/images/setup-clerk/add-application.png)</div>
-
-### Select auth providers
+If you have just created an account for the first time, you will be taken directly to our interactive authentication setup form.
```

Comment:
We should decide if we're going to contract things or not, because it's inconsistent throughout the docs. This sentence says "you will" but the next paragraph says "you'll". I think contractions are better because they sound more natural. Which do you prefer?

```suggestion
If you've just created an account for the first time, you'll be taken directly to the interactive authentication setup form.
```

If you don't like the contractions, we still. need to drop "our" here:

```suggestion
If you have just created an account for the first time, you will be taken directly to the interactive authentication setup form.
```

Reply:
I try my best to remove contractions, as that's what I learned in scientific writing, but maybe it's different for docs. 
The issue with contractions is it can be harder to translate/understand for non-english speakers.

what do you think, though?

Reply:
The translation issue is important enough to avoid contractions, but I've never heard of that before. I tried googling around but can't find anything that confirms or disproves this. I've also never had to avoid contractions writing for any other teams, and Google's docs style guide (not that they're the final word on this) [encourages their writers to use contractions](https://developers.google.com/style/contractions), which I think they'd avoid if they were aware of this problem.

Reply:
yeah if that's how most docs do it, then it's fine. we don't have to worry about them then